### PR TITLE
Configure tsc to emit jsbi.d.ts

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

**Review by jakobkummerow — APPROVED** [review]
Nice, thanks!

Might wanna add "Fixes #89" to the description.

**Comment by mathiasbynens** [comment]
> Might wanna add "Fixes https://github.com/GoogleChromeLabs/jsbi/issues/89" to the description.

Done

File: jsbi.d.ts [hunk 1]
```diff
@@ -1,44 +1,43 @@
...
-  static ADD(x: any, y: any): any;
-  static LT(x: any, y: any): boolean;
-  static LE(x: any, y: any): boolean;
-  static GT(x: any, y: any): boolean;
-  static GE(x: any, y: any): boolean;
-  static EQ(x: any, y: any): boolean;
-  static NE(x: any, y: any): boolean;
+declare class JSBI extends Array {
+    private sign;
+    constructor(length: number, sign: boolean);
```
Comment (12wrigja) [inline jsbi.d.ts:3]: Doesn't this mean someone can now do

```
new JSBI(5, false);
```

and tsc won't complain?

Comment (jakobkummerow) [reply jsbi.d.ts:3]: That sounds likely. What's the best way to prevent that? Does TS allow private constructors? Should we special-case it in the filter?

Comment (12wrigja) [reply jsbi.d.ts:3]: tsc does support marking constructors as private. I'll send a PR to fix this once I'm sure I understand the implications :)

Comment (HolgerJeromin) [reply jsbi.d.ts:3]: Ref #91

**Pull request merged**